pallets/phala/src/compute/computation.rs

```diff
@@ -531,6 +628,26 @@ pub mod pallet {
 			COMPUTING_PALLETID.into_account_truncating()
 		}
 
+		pub fn move_prefix(from_prefix: &[u8], to_prefix: &[u8], max_iterations: u32) {
+			if from_prefix == to_prefix {
+				return
+			}
+		
+			let mut iter = PrefixIterator::<_>::new(from_prefix.clone().into(), from_prefix.into(), |key, value| Ok((key.to_vec(), value.to_vec())));
```

Comment:
I didn't check if this is correct, but if you have tested it on the fork chain, it should be ok.

Reply:
Yes, it is successful done in test net.